[inductor][easy] reuse a single is_aligned function

shunting314 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #108135
* #107722

Resolve comment: https://github.com/pytorch/pytorch/pull/107722#discussion_r1308117422  


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @kadeng @muchulee8 @aakhundov

shunting314 committed:
[inductor][easy] reuse a single is_aligned function

[ghstack-poisoned]

jansel commented on torch/_inductor/codegen/triton_utils.py:
```diff
@@ -37,37 +37,36 @@ def signature_to_meta(signature, *, size_dtype: str):
+        divisible_by_16 = tuple(
+            i
+            for i, arg in enumerate(args)
+            if is_aligned(arg, alignment=ALIGNMENT, include_tensor=True)
```
```suggestion
            if is_aligned(arg, alignment=16, include_tensor=True)
```
given the name of the attribute, I think hardcoding this is fine.

shunting314 committed:
Update on "[inductor][easy] reuse a single is_aligned function"


Resolve comment: https://github.com/pytorch/pytorch/pull/107722#discussion_r1308117422  


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 ipiszy ngimel yf225 chenyang78 kadeng muchulee8 aakhundov

[ghstack-poisoned]

shunting314 commented:
@pytorchbot merge

shunting314 commented:
@pytorchbot label "topic: not user facing"

shunting314 commented:
@pytorchbot merge
